src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf

```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Ennek a netmask értéknek {{ min }} és {{ max }} között kell lennie.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>A fájlnév túl hosszú. {{ filename_max_length }} karakter vagy kevesebb legyen.|A fájlnév túl hosszú. {{ filename_max_length }} karakter vagy kevesebb legyen.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>A jelszó túl egyszerű. Kérjük, használjon egy bonyolultabb jelszót.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Ez az érték olyan karaktereket tratalmaz, amik nem megengedettek.</target>
```

Comment:
Google Translate translates this to: `This value treats characters that are not allowed.`. Can you double check that this is correct?

Reply:
Correctly, this should be translated to: "This value contains characters that are not allowed." I left out the "current restriction-level" part as in the translation it makes no sense to me and seems not relevant.